drivers/internal/SFDP.h

```diff
@@ -67,50 +68,79 @@ struct sfdp_smptbl_info {
     unsigned int erase_type_size_arr[SFDP_MAX_NUM_OF_ERASE_TYPES]; ///< Erase sizes for all different erase types
 };
 
-/** SFDP Parameter Table addresses and sizes */
+/** SFDP JEDEC Parameter Table info */
 struct sfdp_hdr_info {
     sfdp_bptbl_info bptbl;
     sfdp_smptbl_info smptbl;
 };
 
-/** Parse SFDP Headers
- * Retrieves SFDP headers from a device and parses the information contained by the headers
+/** Parse SFDP Database
+ * Retrieves all headers from within a memory device and parses the information contained by the headers
  *
- * @param sfdp_reader Callback function used to read headers from a device
- * @param hdr_info    All information parsed from the headers gets passed back on this structure
+ * Only JEDEC headers are parsed, not vendor specific ones.
  *
- * @return 0 on success, negative error code on failure
+ * @param      sfdp_reader Callback function used to read headers from within a device
+ * @param[out] sfdp_info   Contains the results of parsing the SFDP Database JEDEC headers
+ *
+ * @return MBED_SUCCESS, negative error code on failure
```

Comment:
This is probably obvious from the macro name, but should it still be:
```
* @return MBED_SUCCESS on success, negative error code on failure
```

Reply:
Fixed